src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/PedLoaderStepTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.ac.ebi.eva.pipeline.jobs.steps;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.batch.core.StepContribution;
+import org.springframework.batch.core.StepExecution;
+import org.springframework.batch.core.scope.context.ChunkContext;
+import org.springframework.batch.repeat.RepeatStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.util.ReflectionTestUtils;
+import uk.ac.ebi.eva.pipeline.configuration.CommonConfiguration;
+import uk.ac.ebi.eva.pipeline.configuration.JobOptions;
+
+import static junit.framework.TestCase.assertTrue;
+
+/**
+ * @author Diego Poggioli
+ *
+ * Test for {@link PedLoaderStep}
+ *
+ * Using reflections to inject jobOptions into the tasklet. There are other ways to do this but a been is required.
+ * Otherewise the tasklet needs to be into a Job. See:
+ * http://docs.spring.io/spring-batch/reference/html/testing.html
+ * 10.4 Testing Step-Scoped Components
```

Comment:
Yes we can. In this case however we have a Tasklet and not a Step and this complicates a bit the story because .lunchStep require a Step. Plus we still need a jobLauncher and jobRepository provided by EnableBatchProcessing and Configuration.